test/parallel/test-repl-load-multiline.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 const common = require('../common');
+common.crashOnUnhandledRejection();
```

Comment:
Can you please move this to line 7 blow all the require statements

Reply:
@MylesBorins I have moved this to line 7, thanks to review.